spec/github_helper_spec.rb

```diff
@@ -19,4 +19,27 @@
       end
     end
   end
+
+  describe 'github_token' do
+    after do
+      ENV['GHHELPER_ACCESS'] = nil
+      ENV['GITHUB_TOKEN'] = nil
```

Comment:
I wonder if setting those env vars like that affect the terminal session running the tests? If so, that's not a great practice to have your personal environment modified because you ran a test suite.

Maybe that's not the case (if rspec creates a subshell in which it runs its tests instead of running them directly in the one from which it was invoked), so worth checking.

If it does affect the current shell/terminal session then it's worth finding a better solution, otherwise that means than any GITHUB_TOKEN I have set in my `~/.zshrc` (so I can use `gh`) would be clearer and force me to close and reopen a tab.